### PR TITLE
Send notifications on multisig actions

### PR DESCRIPTION
**Pull request by davecreaser**
Part of CDapp PR: https://github.com/JoinColony/colonyCDapp/pull/3280

### CODE REVIEW

File: src/handlers/multiSig/approvalChanged/approvalChanged.ts [hunk 1]
```diff
@@ -49,4 +58,27 @@ export const handleMultiSigApprovalChanged: EventHandler = async (
...
+
+    if (!multiSigFromDB) {
+      return;
+    }
+
+    const notificationCategory = await getMultisigNotificationCategory(
+      multiSigId,
+    );
+
+    if (notificationCategory && !motion.overallApprovalTimestamp.isZero()) {
```
Comment (bassgeta) [inline src/handlers/multiSig/approvalChanged/approvalChanged.ts:74]: love this solution :100:

**Review by rdig — CHANGES_REQUESTED** [review]
This theoretically works, but couldn't properly test due to JoinColony/colonyCDapp#3280 breaking

**Review by bassgeta — CHANGES_REQUESTED** [review]
Dropped some comments regarding potential improvements, additionally (I've messaged you about this on Discord), but is it necessary for us to get the category for each multisig action, since it seems like the UI is fetching the action on its own, maybe we could skip this step here (unless we may need it in the future, then I'm good with having it in!)
But other than that really nice ways of finding out if a multi-sig approval has passed the threshold, timestamps to the rescue :fire: 
Also I was having dev env issues so for the moment I'm "requesting changes", but once that's gone I'll approve :sunglasses:

File: src/utils/notifications.ts [hunk 2]
```diff
@@ -19,13 +20,24 @@ export enum NotificationCategory {
 }
 
 export enum NotificationType {
```
[the remark this answers is not in the document]
Comment (davecreaser) [reply src/utils/notifications.ts:22]: Yes good thinking! I'll make a note to do just that 🚀

File: src/handlers/multiSig/approvalChanged/approvalChanged.ts [hunk 3]
```diff
@@ -49,4 +58,27 @@ export const handleMultiSigApprovalChanged: EventHandler = async (
       });
     }
   }
+
+  if (multiSigClient) {
+    const motion = await multiSigClient.getMotion(motionId);
+    const multiSigFromDB = await getMultiSigFromDB(multiSigId);
```
[the remark this answers is not in the document]
Comment (davecreaser) [reply src/handlers/multiSig/approvalChanged/approvalChanged.ts:64]: Good idea, done :)

**Review by bassgeta — APPROVED** [review]
Yup, everything is still legit and we got rid of 1 more query, `l e g i t` :sunglasses: 
Tested as part of its CDapp counterpart and can confirm it all works like a well oiled machine :ship:

**Review by rdig — APPROVED** [review]
Tested as part of JoinColony/colonyCDapp#3280 and working as intended. 

All good.

**Review by iamsamgibbs — APPROVED** [review]
Reviewed with CDapp counterpart

**Pull request merged**